tests/js/setup.ts

```diff
@@ -189,6 +189,21 @@ const routerFixtures = {
     ...params,
   }),
 
+  routeComponentProps: (
+    params: Partial<RouteComponentProps<any, any>> = {}
```

Comment:
would be kinda nice to use a generic here since which was my thinking here https://github.com/getsentry/sentry/blob/master/tests/js/sentry-test/initializeOrg.tsx#L35 since you get type checking pretty easily

Reply:
Hmm, that causes a typeerr on params, I see you had to cast that in your change. Since that exists already I'll do the cast too